src/api-reference/fi-service/financial-integration-service.markdown

```diff
@@ -0,0 +1,1124 @@
+# Financial Integration Service
+
+* [Financial Integration API](#fis)
+  - [GET a List of Financial Transactions based on a search criteria](#gettransactions)
+  - [POST Financial Transactions acknowledgements](#postacknowledgements)
+  - [POST Financial Transactions confirmations](#postconfirmations)
+* [Response Codes](#responsecodes)
+* [Financial Integration sample documents](#examples)
+  - [Expense sample document](#expense)
+  - [Invoice sample document](#invoice)
+  - [Cash Advance sample document](#cashadvance)
+
+## Financial Integration API
+
+The Financial Integration API allows an external system to interact with financial documents generated from Concur, for financial posting into an ERP.
+
+This Web Service provides an automated solution to request available data objects such as approved expense reports, cash advance requests and invoices, to import to the client internal system, with an opportunity to send posting confirmation back into Concur before the object is locked down and cannot be altered in Concur.
+
+Below are some benefits for using the Financial Integration service:
+
+* Using a modern method, no more handling of flat files.
+* Close to real time experience VS SAP Concur over night processing.
+* 1 to 1 Concur to ERP reconciliation VS the extract batch processing.
+* Opportunity to modify and retry before the object is locked in Concur.
+* Insures data sync between systems.
+
+### Version
+
+1.0
+
+### Authorization
+
+The Financial Integration API works only with the new [Authentication API](https://developer.concur.com/api-reference/authentication/apidoc.html).
+
+```
+HEADER:   "Authorization: Bearer {YOUR ACCESS TOKEN}"
```

Comment:
Remove the term 'HEADER' and extraneous whitespace; stick with the simple output:

Reply:
```
Authorization: Bearer {token}"
```

Reply:
Take a look at the revised version Imene submitted today addressing your input (last commit 991304a8). For the entityID and CompanyOutask ID, Teresa’s team are working on a v2 of this API to hide those details to the customers.